volatility3/framework/plugins/windows/mbrscan.py

```
@@ -0,0 +1,241 @@
+                            partition_table.ThirdEntry.get_starting_cylinder(),
+                            partition_table.ThirdEntry.get_starting_chs(),
+                            partition_table.ThirdEntry.get_starting_sector(),
+                            partition_table.ThirdEntry.get_ending_cylinder(),
+                            partition_table.ThirdEntry.get_ending_chs(),
+                            partition_table.ThirdEntry.get_ending_sector(),
+                            format_hints.Hex(partition_table.ThirdEntry.get_size_in_sectors()),
+                            partition_table.FourthEntry.is_bootable(),
+                            format_hints.Hex(partition_table.FourthEntry.get_bootable_flag()),
+                            partition_table.FourthEntry.get_partition_type(),
+                            format_hints.Hex(partition_table.FourthEntry.PartitionType),
+                            format_hints.Hex(partition_table.FourthEntry.get_starting_lba()),
+                            partition_table.FourthEntry.get_starting_cylinder(),
+                            partition_table.FourthEntry.get_starting_chs(),
+                            partition_table.FourthEntry.get_starting_sector(),
+                            partition_table.FourthEntry.get_ending_cylinder(),
+                            partition_table.FourthEntry.get_ending_chs(),
+                            partition_table.FourthEntry.get_ending_sector(),
+                            format_hints.Hex(partition_table.FourthEntry.get_size_in_sectors()),
+                            interfaces.renderers.Disassembly(bootcode, 0, architecture),
+                            format_hints.HexBytes(bootcode)
+                        ))
+                else:
+                    vollog.log(constants.LOGLEVEL_VV, f"Not a valid MBR: Data all zeroed out : {format_hints.Hex(offset)}")
+            
+            except exceptions.PagedInvalidAddressException:
+                pass
```

I'd prefer that an exception had a log at the highest level (like `LOGLEVEL_VVVV` so it only shows up when someone really wants to see it), or that a proper test be put in place if this is a thing we expect to run into often.  If it really is an exceptional circumstance I'd prefer not to just ignore the error and move on.